crawler/src/main/java/com/brandwatch/minibcr/crawler/service/CrawlerService.java

```diff
@@ -0,0 +1,14 @@
+package com.brandwatch.minibcr.crawler.service;
+
+public class CrawlerService implements Crawler {
```

Comment:
What's the point of this service?

Reply:
To incorporate delegation. My thinking was like Crawler can be implemented by "RedditCrawler" / "TwitterCrawler" etc. this class delegates accordingly for client based on the ref., any further enhancing around the method can be done here. Technically can get this done without using this delegation. Please let me know if you think this is not required.

Reply:
I think what confuses me is that it implements Crawler.